sdk/python/touca/_options.py

```diff
@@ -94,6 +97,33 @@ def _validate_options(existing: dict):
         raise ValueError(f"missing required option(s) {','.join(keys)}")
 
 
+def _get_api_key_and_url() -> Union[
+    Tuple[str, str],
+    None,
+]:
+    from os import environ
+    from sys import exit
+
+    # 1. get environment variables
+    api_key = environ.get("TOUCA_API_KEY", None)
+    api_url = environ.get("TOUCA_API_URL", None)
+
+    # 2. get config file
+    if not api_key or not api_url:
+        config = config_file_parse()
+        if config and config.has_section("settings"):
+            api_key = config.get("settings", "api-key", fallback=api_key)
+            api_url = config.get("settings", "api-url", fallback=api_url)
+
+    if api_key is None or api_url is None:
+        logger.error(
+            "api-key or api-url is missing, "
+            "for more information see https://touca.io/docs/cli/config"
+        )
+        exit(1)
+    return api_key, api_url
+
+
```

Comment:
I am not quite convinced that this function is needed. Could you elaborate why we couldn't just use the existing `update_options` function? Perhaps if we un-comment line 75, it would do exactly what we need it to.